go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openfga/cli
 
-go 1.22.7
+go 1.23.0
+
+toolchain go1.24.0
```

Comment:
Any reason to not use 1.24.0 as the version on line 3?

Reply:
Not sure if any are still using it like so, but previously we had community members importing parts of the CLI in their code.

So maintaining the same 2 major version support, while not critical, seems OK - it follows the same policy as language and the Go SDK and OpenFGA